recipe/meta.yaml

```diff
@@ -2,7 +2,11 @@
 {% set name_suffix = "3.6" %}
 {% endif %}
 {% set name = "pypy" ~ name_suffix %}
+{% if name_suffix == "3.6" %}
 {% set version = "7.3.3" %}
+{% else %}
+{% set version = "7.3.4" %}
+{% endif %}
```

Comment:
Since there are no 3.6 builds for 7.3.4, we can just do.
```suggestion
{% set version = "7.3.4" %}
```

If we want to rebuild 7.3.3 for some reason, we can create a new git branch.

Reply:
Cool. Would it be better to let this CI run finish to see what else is broken or to commit this now and restart?

Reply:
Either is fine with me. Your call.

Reply:
CI failed as I feared, unable to apply patches. I will try them locally. This change simplifies things, there will be no conflict which patches to apply.